querydsl-libraries/pom.xml

```diff
@@ -138,18 +138,18 @@
           </execution>
         </executions>
       </plugin>
-      <plugin>
-        <groupId>com.github.siom79.japicmp</groupId>
-        <artifactId>japicmp-maven-plugin</artifactId>
-        <executions>
-          <execution>
-            <goals>
-              <goal>cmp</goal>
-            </goals>
-            <phase>verify</phase>
-          </execution>
-        </executions>
-      </plugin>
+<!--      <plugin>-->
```

Comment:
leave the plugin here, do not change change anything related to japicmp on parent poms